terraform/groups/ecs-service/data.tf

```diff
@@ -42,6 +42,15 @@ data "aws_lb_listener" "service_lb_listener" {
   port = 443
 }
 
+data "aws_lb" "secondary_lb" {
+  name = "${var.environment}-chs-apichgovuk-private" # test attach to private version of cidev main API
```

Comment:
no need for the comment :)

Reply:
I didn't think so but thought I'd leave it in and see what happened :)

Removed